src/api/wishlist.js

```diff
@@ -0,0 +1,27 @@
+import Base from './base';
+
+export default class extends Base {
+    /**
+     * @Constructor
+     */
+    constructor(version) {
+        // call parent
+        super(version);
+
+        // set up class variables
+        this.endpoint = '/wishlist.php?action=add&wishlistid=[WISHLIST_ID]&product_id=[PRODUCT_ID]';
```

Comment:
if we are going to use `this.endpoint` property, it should have only` this.endpoint  = '/wishlist.php'` to be a reusable in other methods as a base url for the entity. Since it's only one endpoint, you would recommend to omit the **constructor** at all in this class, so the parent constructor will be called directly.

And then url could look like
`const url = `/wishlist.php?action=add&wishlistid=${wishlistId}&product_id=${productId}``

Reply:
@jairo-bc done!